zerver/openapi/zulip.yaml

```diff
@@ -3305,15 +3305,6 @@ paths:
 
                                         **Changes**: New in Zulip 4.0 (feature level 33). Moderators option added in
                                         Zulip 4.0 (feature level 62).
-                                    default_language:
-                                      type: string
-                                      description: |
-                                        The default UI language for new users joining this organization.
-                                    default_twenty_four_hour_time:
-                                      type: boolean
-                                      description: |
-                                        Whether new members of this organization will see times displayed in
-                                        24-hour time (true) or 12-hour time (false).
```

Comment:
One strategic question we should potentially talk about is whether we want to migrate everything that is presently in the `update` format to use `update_dict`, since that's the more generic format; but that'll require some coordination with clients in the "api design" stream.  So I think it's correct to merge this and think about that later.